modules/nw-operator-cr.adoc

```diff
@@ -259,7 +259,42 @@ ifdef::operator[]
 If the field is present, IPsec is enabled for the cluster.
 endif::operator[]
 
+|`policyAuditConfig`
+|`object`
+|Specify a configuration object for customizing network policy audit logging. If unset the defaults audit log settings are used.
```

Comment:
Comma after `If unset`?

I <heart> the "The default value is" repetition in the following table.

Reply:
@mikemckiernan, I try to learn new tricks at the most unexpected intervals.